dirty2.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+)
+
+func zFoo(arg string) int {
```

Comment:
__[staging][gometalinter]__
* [deadcode] zFoo is unused

<!-- Comment by [SideCI](https://sideci.com) -->